### DISCUSSION TITLE
Error, no templates were found.

### DISCUSSION THREAD

**Discussion by 0xhunster** [projectdiscovery/nuclei#774] [Q&A] [answered]
**Issue**
I saw nuclei template is there, but when I tried to scan with that template, nuclei showed "Error, no templates were found." This is my issue.

**Nuclei version**
I used the Nuclei version v2.3.7.

**Screenshot of the error or bug**
![16736](https://user-images.githubusercontent.com/46501627/122170707-b4e61980-cea0-11eb-99ff-0ede10fe7b5c.png)

**Comment by ehsandeep** [comment, marked as answer]
@0xAkashsec that error indicates, something is wrong with template syntax, you can see the detailed error with `-v` flag.
